### DISCUSSION TITLE
Is there a way to remove the new ‘You’ tab and make it back to what it was?

### DISCUSSION THREAD

**Discussion by Shaukariya** [arichornlover/uYouEnhanced#191] [Q&A]
Title basically.

**Comment by arichornlover** [comment]
Yes. But sadly the only way to get the Library Tab back is using the App Version Spoofer found in uYouPlus > Miscellaneous

**Reply by Shaukariya** [reply]
Tried that didn't work. Which version should I try?

**Reply by arichornlover** [reply]
try `v18.34.5`, and once you do that please restart the YouTube app 2-3 times. or if this didn’t work then keep picking lower versions.
Since the “You” Tab is server-side, the version spoofer will undo it and bring back the Library Tab.
